packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -114,6 +115,134 @@ describe("Garbage Collection Tests", () => {
+            it("No metadata", () => {
+                const gc = createGcWithPrivateMembers({});
+                assert(!gc.gcEnabled, "gcEnabled incorrect");
+                assert(!gc.sweepEnabled, "sweepEnabled incorrect");
+                assert(gc.sessionExpiryTimeoutMs === undefined, "sessionExpiryTimeoutMs incorrect");
+                assert(gc.sweepTimeoutMs === undefined, "sweepTimeoutMs incorrect");
+                assert.equal(gc.latestSummaryGCVersion, 0, "latestSummaryGCVersion incorrect");
+            });
+            it("gcFeature 0", () => {
+                const gc = createGcWithPrivateMembers({ gcFeature: 0 });
+                assert(!gc.gcEnabled, "gcEnabled incorrect");
+                assert.equal(gc.latestSummaryGCVersion, 0, "latestSummaryGCVersion incorrect");
+            });
+            it("gcFeature 0, sweepEnabled true", () => {
+                const gc = createGcWithPrivateMembers({ gcFeature: 0, sweepEnabled: true });
+                assert(!gc.gcEnabled, "gcEnabled incorrect");
+                assert(gc.sweepEnabled, "sweepEnabled incorrect");
+                assert.equal(gc.latestSummaryGCVersion, 0, "latestSummaryGCVersion incorrect");
+            });
+            it("gcFeature 1", () => {
+                const gc = createGcWithPrivateMembers({ gcFeature: 1 });
+                assert(gc.gcEnabled, "gcEnabled incorrect");
+                assert.equal(gc.latestSummaryGCVersion, 1, "latestSummaryGCVersion incorrect");
+            });
+            it("sweepEnabled false", () => {
+                const gc = createGcWithPrivateMembers({ sweepEnabled: false });
+                assert(!gc.sweepEnabled, "sweepEnabled incorrect");
+            });
+            it("sessionExpiryTimeoutMs set", () => {
+                const gc = createGcWithPrivateMembers({ sessionExpiryTimeoutMs: 123 });
+                assert.equal(gc.sessionExpiryTimeoutMs, 123, "sessionExpiryTimeoutMs incorrect");
+            });
+            it("Metadata Roundtrip", () => {
+                const inputMetadata: IGCMetadata = { sweepEnabled: true, gcFeature: 1, sessionExpiryTimeoutMs: 123 };
+                const gc = createGcWithPrivateMembers(inputMetadata);
+                const outputMetadata = gc.getMetadata();
+                assert.deepEqual(outputMetadata, inputMetadata, "getMetadata returned different metadata than loaded from");
+            });
+        });
+
+        describe("New Container", () => {
+            beforeEach(() => {
+                // default Session Expiry Enabled
+                injectedSettings[runSessionExpiryKey] = true;
+            });
+            it("No options", () => {
+                const gc = createGcWithPrivateMembers(undefined /* metadata */, {});
+                assert(gc.gcEnabled, "gcEnabled incorrect");
+                assert(!gc.sweepEnabled, "sweepEnabled incorrect");
+                assert(gc.sessionExpiryTimeoutMs !== undefined, "sessionExpiryTimeoutMs incorrect");
```

Comment:
GC session expiry should be undefined by default

Reply:
Probably due to me setting it by default in beforeEach.  Reverted that so it's explicit in each test.